scripts/gluster-block/setup-gluster-block-glusto.yml

```diff
@@ -206,6 +205,9 @@
     - python-pip
     - json-c-devel
 
+  - name: Copy rpm file to server
+    command: rpm -i http://mirror.centos.org/centos/7/storage/x86_64/gluster-3.12/glusterfs-devel-3.12.9-1.el7.x86_64.rpm
```

Comment:
This is not a good idea. What are the documented reasons that we need special snowflake API package?

Reply:
Yes, this is not a good idea. Some API changes have happened and Prasanna will be sending a patch to resolve the build issues. Having this packages will resolve the build issues for now and we can go ahead with testing the CI. Once the patch is sent, I will remove this from the playbook as we won't be needing it then.